source/common/router/config_impl.h

```diff
@@ -474,13 +474,15 @@ class ShadowPolicyImpl : public ShadowPolicy {
   const std::string& runtimeKey() const override { return runtime_key_; }
   const envoy::type::v3::FractionalPercent& defaultValue() const override { return default_value_; }
   bool traceSampled() const override { return trace_sampled_; }
+  bool disableShadowHostSuffixAppend() const override { return disable_shadow_host_suffix_append_; }
 
 private:
   const std::string cluster_;
   const Http::LowerCaseString cluster_header_;
   std::string runtime_key_;
   envoy::type::v3::FractionalPercent default_value_;
   bool trace_sampled_;
+  bool disable_shadow_host_suffix_append_;
```

Comment:
nit: consider converting to const, and maybe moving the initialization to the c'tor's MIL.

Reply:
So want me to add it to `RequestMirrorPolicy` in the MIL or as another parameter to the MIL?